README.md

```diff
@@ -1,64 +1,41 @@
-IBM_DB_SA
+SQLAlchemy-IBMi
 =========
 
-The IBM_DB_SA adapter provides the Python/SQLAlchemy interface to IBM Data Servers.
+The SQLAlchemy-IBMi adapter provides the Python/SQLAlchemy interface to IBM i db2.
 
 Version
 --------
-0.3.5 (2019/05/30)
+0.1 (2019/11/26)
```

Comment:
Let's just leave this section blank, since we haven't released any versions yet.